ale_linters/cloudformation/cfn_python_lint.vim

```diff
@@ -4,22 +4,22 @@
 function! ale_linters#cloudformation#cfn_python_lint#Handle(buffer, lines) abort
     " Matches patterns line the following:
     "
-    " sample.template.yaml:96:7:96:15: [E3012] Property Resources/Sample/Properties/FromPort should be of type Integer
-    let l:pattern = '\v^(.*):(\d+):(\d+):(\d+):(\d+): \[([[:alnum:]]+)\] (.*)$'
+    " sample.template.yaml:96:7:96:15:E3012:Property Resources/Sample/Properties/FromPort should be of type Integer
+    let l:pattern = '\v^(.*):(\d+):(\d+):(\d+):(\d+):([[:alnum:]]+):(.*)$'
```

Comment:
Has the output changed in a newer version of the linters? If so, put both the old and new formats in the tests, and ALE should support both, so both older and newer versions work.

Reply:
Yes it has changed to a more parsable output from 0.2.2 onwards and should be considered stable now.

Given the relative age of the project (~2 months) I am not sure it is worthwhile to support the old format, but I can try to make it work if you want. Do you know from the top of your head another linter in ALE that does this?

Reply:
There are many linters where both the older and newer formats are supported, but I didn't realise the project was so young. I think it's fine to just switch to the new format at this stage, then.